# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pilot-link/bluez-configure.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

===================================================================
RCS file: /cvs/pilot-link/pilot-link_src/configure.ac,v
retrieving revision 1.58
retrieving revision 1.59
diff -u -r1.58 -r1.59
--- ./configure.ac	2007/02/15 01:01:28	1.58
+++ ./configure.ac	2007/02/15 02:18:42	1.59
@@ -350,8 +350,9 @@
 
 AC_MSG_CHECKING([for BlueZ bluetooth support])
 AC_ARG_WITH(bluez,
-                AC_HELP_STRING([--with-bluez],
-                        [Enable usage of BlueZ]),,
+                AC_HELP_STRING([--with-bluez], 
+			[Enable usage of BlueZ]), 
+			use_bluez="${withval}",
                 use_bluez=auto)
 if test "x$use_bluez" != "xno"; then
         if test "x$have_bluez" = xno && test "x$use_bluez" = xyes; then
